test/launch.js

```diff
@@ -103,4 +101,13 @@ describe("nearleyc", function() {
         var crontabResults = read('test/classic_crontab.results');
         parse(classicCrontab, crontabTest).should.deep.equal([JSON.parse(crontabResults)]);
     });
+
+    it('parentheses', function() {
+        // Try compiling the grammar
+        var parentheses = compile(read("examples/parentheses.ne"));
+        var parenthesesTest1 = '[(((<>)()({})())(()())(())[])]';
+        var parenthesesTest2 = '<<[([])]>([(<>[]{}{}<>())[{}[][]{}{}[]<>[]{}<>{}<>[]<>{}()][[][][]()()()]({})<[]>{(){}()<>}(<>[])]())({})>';
+        parse(parentheses, parenthesesTest1).should.deep.equal([true]);
+        parse(parentheses, parenthesesTest2).should.deep.equal([true]);
```

Comment:
Is it possible to make this test match something more interesting? We want to make sure here that nearley is working as expected. 

(A valid answer here would be "we're just testing the recogniser here and not the parser". I would be fine with that. :) )